src/aliceVision/geometry/Pose3.hpp

```diff
@@ -9,85 +9,108 @@
 
 #include <aliceVision/numeric/numeric.hpp>
 #include <boost/math/constants/constants.hpp>
+#include <aliceVision/geometry/lie.hpp>
+
 
 namespace aliceVision {
 namespace geometry {
 
-// Define a 3D Pose as a 3D transform: [R|C] t = -RC
+// Define a 3D Pose as a SE3 matrix
 class Pose3
 {
   protected:
-    Mat3 _rotation;
-    Vec3 _center;
+    SE3::Matrix _homogeneous;
 
   public:
-    // Constructors
-    Pose3() : _rotation(Mat3::Identity()), _center(Vec3::Zero()) {}
-    Pose3(const Mat3& r, const Vec3& c) : _rotation(r), _center(c) {}
-    Pose3(const Mat34& Rt)
-    : _rotation(Rt.block<3,3>(0,0))
+
+    Pose3() : _homogeneous(SE3::Matrix::Identity())
+    {
+
+    }
```

Comment:
prefer member initialization and default constructor:
```c++
protected:
    SE3::Matrix _homogeneous{SE3::Matrix::Identity()};

public:
     Pose3() = default;
```

Reply:
Done